src/workflows/contrib/start_service.py

```diff
@@ -82,6 +82,32 @@ def run(
             help="Name of the service to start. Known services: "
             + ", ".join(known_services),
         )
+        parser.add_option(
+            "-l",
```

Comment:
```suggestion
```
I don't think we need a short option for this, it should only be necessary when we explicitly want it